### PR TITLE
Update call-park-and-retrieve.md

### PR DESCRIPTION
**Pull request by lindspea**
https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/1998
Removed wording.

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/call-park-and-retrieve.md [hunk 1]
```diff
@@ -51,7 +51,7 @@ Call park and retrieve is currently supported by the following clients and devic
 
 ## Configuring call park and retrieve
 
-You must be an administrator to configure call park and retrieve, and the feature is disabled by default. You can enable it for users and create user groups using the call park policy. When you apply the same policy to a set of users, they can park and retrieve calls among themselves. To configure call park for users and create call park user groups, follow the [Assign a call park policy](#assign-a-call-park-policy) procedure below.
+You must be an administrator to configure call park and retrieve, and the feature is disabled by default. You can enable it for users and create user groups using the call park policy. When you apply the same policy to a set of users, they can park and retrieve calls among themselves. To configure call park for users and create call park user groups, follow the [Assign a call park policy](#assign-a-call-park-policy).
```
Comment (nenonix) [inline Teams/call-park-and-retrieve.md]:
```suggestion
You must be an administrator to configure call park and retrieve, and the feature is disabled by default. You can enable it for users and create user groups using the call park policy. When you apply the same policy to a set of users, they can park and retrieve calls among themselves. To configure call park for users and create call park user groups, follow the [Assign a call park policy](#assign-a-call-park-policy) steps below.
```

**Comment by scanum** [comment]
@LolaJacobsen Hi Lola, changes look correct. Could you please confirm and merge? Thanks!

**Comment by tonysmit** [comment]
These changes aren't needed. I worked with Sharon to write it so it's fine. "procedure below" isn't needed. The procedure that they removed is correct. "To assign a policy to one or more users....", they must use this procedure and not the one they added. Thanks.

**Pull request closed without merging**